android/build.gradle

```diff
@@ -26,6 +26,7 @@ buildscript {
         classpath("com.android.tools.build:gradle:7.1.1")
         classpath("com.facebook.react:react-native-gradle-plugin")
         classpath("de.undercouch:gradle-download-task:5.0.1")
+        classpath 'com.google.gms:google-services:4.3.13'
```

Comment:
This is completely fine though, but I think it would be a good idea to follow the same pattern as in above dependencies 🤓 
```
classpath("com.google.gms:google-services:4.3.13")
```

Reply:
Thanks for the feedback but does that make any difference in functionality ?

Reply:
No it doesn't make any difference. Both syntax are supported by gradle files, so functionality will remain same.